### PR TITLE
JavaScript: Mobile menu

### PR DESCRIPTION
**Pull request by GutemaG**
## Adding Mobile Menu using JavaScript 😄 

- Add JavaScript linter
- Add mobile menu 📱 
-  Use JavaScript to toggle and remove classes from an element 👍 
-  Add style for mobile menu

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,15 +15,16 @@
...
   <body>
     <header class="header">
-      <nav>
+      <nav class="">
         <a href="#" class="logo">My Logo</a>
         <button class="menu-icon">
-          <img src="icons/menu-icon.svg" alt="menu-icon" class="" >
+          <img src="icons/menu-icon.svg" alt="menu-icon" class="open-menu" >
         </button>
         <ul class="menu">
```
Comment (topeogunleye) [inline index.html:23]: - [ ] I kindly suggest that you add the background blur that show when the mobile-menu is active. you can do that by inserting and removing css with javascript.

```
.body {
   backdrop-filter: blur(5px);
}
```




| Your Design  | Figma Design |
| ------------- | ------------- |
| image | image |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @GutemaG and @ShinhyoBelok, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-  Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (GutemaG) [reply index.html:23]: Hello, @topeogunleye I tried to make its background blur, but I don't know what is wrong it is not working, can you help me or suggest something to read?

Comment (topeogunleye) [reply index.html:23]: - [ ] You can add this class on your main element container to blur the whole background. Use Javascript to add and remove the 'bured' css class applied.

.blured {
  filter: blur(2px);
  -webkit-filter: blur(2px);
}

Comment (GutemaG) [reply index.html:23]: Thank you again,@topeogunleye , What do you think about this design?
image

**Comment by GutemaG** [comment]
## Correct some CSS from first code reviewer
 - Add `backdrop: blur(5px)` to style

**Review by christianbiring1 — APPROVED** [review]
Hi @GutemaG , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
imageswelljob

## Highlights
- [ ] All linters checks are green :+1: 
- [ ] Good commit message
- [ ] Descriptive PR `summary`
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by topeogunleye** [comment]
> ## Correct some CSS from first code reviewer
> * Add `backdrop: blur(5px)` to style

Great Job increasing the blur on the background

File: index.html (hunk 1 above)
Comment (topeogunleye) [reply index.html:23]: @GutemaG Great Job increasing the blur on the background

**Pull request merged**